<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qmain_window</class>
 <widget class="QWidget" name="Qmain_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>qtCRMInterface</string>
  </property>
  <widget class="QLabel" name="Qlabel_title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CRM brought to you by The JoestarsÂ®</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Qtable">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>101</y>
     <width>1061</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeWidget" name="Qlist_category">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>60</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="Qline_main">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>10</y>
     <width>31</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>154</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Qvert_layout_action">
    <item>
     <widget class="QGroupBox" name="Qbox_general">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Actiuni Generale</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="Qvert_layout_general">
        <item>
         <widget class="Line" name="Qline_general">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_general_email">
          <property name="text">
           <string>Send E-mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_general_comp">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Vezi info comp.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_general_1">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Button 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_general_2">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Button 2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Qbox_client">
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Actiuni Client</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>135</width>
         <height>150</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="Qvert_layout_client">
        <item>
         <widget class="Line" name="Qline_client">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_client_add">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Adaugare intrare</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_client_modif">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Modifica intrare</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_client_del">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Stergere intrare</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_client_1">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Button 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Qbutton_client_2">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Button 2</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="Qtest_search">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>60</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
